common-gradle-enterprise-bazel-configuration/.bazelrc

```diff
@@ -0,0 +1,20 @@
+# To publish Build Scans to your Develocity server, add the following to the .bazelrc file in your 
+# build workspace:
+common:develocity --remote_cache=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --bes_results_url=https://<<develocity-server>>/build/ # adjust to your Develocity server
+common:develocity --bes_backend=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --build_event_publish_all_actions=true
+common:develocity --noslim_profile
+
+build --config=develocity
+query --config=develocity
```

Comment:
Where does the project name get set?

Reply:
issue has been fixed

Reply:
does that set the project name or the project id?

Reply:
following [Setting the project name](https://docs.gradle.com/enterprise/bazel-configuration/#project-name) it sets the project name, but in the next section https://docs.gradle.com/enterprise/bazel-configuration/#project-level-access-control it states the same properties are used for the project id

Reply:
Can you remove it for now? Following an internal discussion, it looks like there is no way to set the project name.

Reply:
removed